coderd/database/dbfake/dbfake.go

```diff
@@ -320,6 +327,8 @@ func (t TemplateVersionBuilder) Do() TemplateVersionResponse {
 	t.seed.OrganizationID = takeFirst(t.seed.OrganizationID, uuid.New())
 	t.seed.ID = takeFirst(t.seed.ID, uuid.New())
 	t.seed.CreatedBy = takeFirst(t.seed.CreatedBy, uuid.New())
+	// nolint: revive
+	t.fileID = takeFirst(t.fileID, uuid.New())
```

Comment:
heads-up: the linter has started complaining about this, we may want to consider squelching this rule for this particular case.

Reply:
What rule is the problem?

Reply:
https://github.com/mgechev/revive/blob/master/RULES_DESCRIPTIONS.md#modifies-value-receiver